src/snmp/Request.h

```diff
@@ -33,7 +33,7 @@ class Request: public Ipc::Request
     virtual Pointer clone() const;
 
 private:
-    Request(const Request& request);
+    Request(const Request& request) = default;
```

Comment:
IIRC you suggested removing this explicit declaration, but should we allow the outer code to copy objects, thus disregarding clone()?

Reply:
I believe we were discussing the parent class at the time. I would probably keep _this_ constructor private to force folks to use clone(). If nothing else, this would be an out-of-scope change AFAICT.

Disclaimer: I have not studied why the code wants to force folks to use clone().